docker-development/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+---
+version: '3.8'
+networks:
+  default:
+    driver: host
```

Comment:
This throws

```
 ✘ Network docker-development_default  Error                                                              0.0s 
failed to create network docker-development_default: Error response from daemon: only one instance of "host" network is allowed
```
On MacOS since it's only supported on Linux.

Let's go with default network for now and mention any possible caveats connected to the decision.